### PR TITLE
feat: expose version object in releases

### PR DESCRIPTION
**Pull request by DevVersion**
* In releases there will be a constant called `VERSION` that holds the current version of the installed package (material or cdk)
* This is similar as for every @angular package like core, forms, compiler.

### CODE REVIEW

**Comment by jelbourn** [comment]
Before we merge this, I need to talk to @vikerman about how this version gets updated for Angular doing the sync into Google.

**Comment by DevVersion** [comment]
Rebased.

**Comment by ThomasBurleson** [comment]
I love this PR. Need this in Flex-Layout!

**Comment by DevVersion** [comment]
@jelbourn Any updates on this one?

**Comment by jelbourn** [comment]
Every time I remember it @vikerman is away from his desk.

**Comment by jelbourn** [comment]
Finally followed up on this- no action needed on your part. Will be good to merge after my cdk entry-points PR (version will be the only thing under `@angular/cdk` directly)

**Comment by DevVersion** [comment]
@jelbourn Rebased. Please have another look.

File: tools/package-tools/version-placeholders.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import {readFileSync, writeFileSync} from 'fs';
```
Comment (DevVersion) [inline tools/package-tools/version-placeholders.ts]: Good question. Looks like the old one before the build changes is still there after rebasing.

Comment (DevVersion) [inline tools/package-tools/version-placeholders.ts]: Done.

File: tools/package-tools/version-placeholders.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    .filter(String);
+
+  // Walk through every file that contains version placeholders and replace those with the current
+  // version of the root package.json file.
+  files.forEach(filePath => {
+    let fileContent = readFileSync(filePath, 'utf-8');
+
+    fileContent = fileContent.replace(versionPlaceholderRegex, buildConfig.projectVersion);
+
+    writeFileSync(filePath, fileContent);
```
Comment (jelbourn) [inline tools/package-tools/version-placeholders.ts]: Does this `writeFileSync` need `'utf-8'`?

Comment (DevVersion) [inline tools/package-tools/version-placeholders.ts]: `writeFileSync` always defaults to `UTF-8` as encoding ([reference](https://nodejs.org/api/fs.html#fs_fs_writefilesync_file_data_options))

**Pull request merged**